### PR TITLE
Fix data creation script in on-demand environment

### PR DESCRIPTION
**Pull request by chmanie**
## Description

This PR aims to fix the creation of temporary data in the on-demand development environment.

I also found bugs in two lambdas, as they were missing the `cross-fetch` polyfill.

### CODE REVIEW

**Review by bassgeta — APPROVED** [review]

**Review by jakubcolony — COMMENTED** [review]
Hey @chmanie, left you a question regarding the cross-fetch polyfill

**Review by ArmandoGraterol — APPROVED** [review]
LGTM 👍

File: amplify/backend/function/qaSSMtest/src/index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require('cross-fetch/polyfill');
```
[the remark this answers is not in the document]
Comment (chmanie) [reply amplify/backend/function/qaSSMtest/src/index.js:1]: Yeah, it actually is but only when on node 20 for some reason. This PR is already part of the nodeJS 20 upgrade, so we can just close it.

**Comment by chmanie** [comment]
This PR is already part of #1714, so we can close it

**Pull request closed without merging**